packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -146,6 +144,10 @@ export class Telemetry {
 		}
 	}
 
+	reset() {
```

Comment:
There is also a call to `rudderStack.reset` in `identify` action ([here](https://github.com/n8n-io/n8n/blob/8fd044eae03fa314a7ab35a16967dab09499c888/packages/editor-ui/src/plugins/telemetry/index.ts#L99)). Do we still need that?

Reply:
we still need that when initializing RS if user is not logged in.